src/components/ui/Link/link.scss

```
@@ -1,48 +1,96 @@
 @import "src/styles/scaffold";
 
-a, [data-fs-link] {
-  border-radius: var(--fs-border-radius-default);
-  transition: box-shadow .5s ease;
+[data-fs-link] {
+  // --------------------------------------------------------
+  // Design Tokens for Link
+  // --------------------------------------------------------
 
-  @include focus-ring-visible;
+  // Default properties
+  --fs-link-min-width                  : auto;
+  --fs-link-min-height                 : var(--fs-link-min-width);
+  --fs-link-padding                    : var(--fs-spacing-2) var(--fs-spacing-0);
+
+  --fs-link-border-radius              : var(--fs-border-radius-default);
+
+  --fs-link-text-color                 : var(--fs-color-link);
+  --fs-link-text-line-height           : 1.5;
+  --fs-link-text-decoration            : none;
+  --fs-link-text-decoration-hover      : underline;
+  --fs-link-text-color-visited         : var(--fs-color-link-visited);
```

Moves `--fs-link-text-color-visited` next to `--fs-link-text-color`
```suggestion
  --fs-link-text-color                 : var(--fs-color-link);
  --fs-link-text-color-visited         : var(--fs-color-link-visited);
  --fs-link-text-line-height           : 1.5;
  --fs-link-text-decoration            : none;
  --fs-link-text-decoration-hover      : underline;
```